README.md

```diff
@@ -117,6 +117,17 @@ dev_setup.sh
 dev_setup.bat
 ```
 
+#### Environment
+
+Populate the following 3 `.env` files with the relevant urls for your GAEN server:
+
+```
+.env.bt
+.env.bt.release
+```
+
+**Note:** Members of the `Path-Check` org can complete this step by running `yarn set-ha` and passing in the 2-letter ha abbreviation as the first argument (i.e. `yarn set-ha pc`)
```

Comment:
Questions for my understanding:
1. is "pc" HA "path check" here? 
2. What will be thee strategy for knowing what HAs are available?

Reply:
1. Yes, "pc" HA  is "pathcheck" - this configuration will ideally point to its own gaen server instance in the future  
2. I'm thinking that information should be private (but stored in place where all members of the Path Check org know where to look for it). At the moment developers can look here https://github.com/Path-Check/pathcheck-mobile-resources/tree/master/environment